tests/conftest.py

```diff
@@ -1,17 +1,15 @@
-import mock
 import pytest
 
+from pyramid import testing
 from pyramid_zipkin import request_helper
+from pyramid.testing import DummyRequest
 
 
 @pytest.fixture
 def dummy_request():
-    request = mock.Mock()
```

Comment:
I thought it would be better to use Pyramid's dummyRequest because otherwise, you would need to mock request.registry to also support dictionary behavior (request.registry['version']) while also supporting request.registry.settings.